core/src/main/scala/cats/ApplicativeError.scala

```diff
@@ -104,6 +104,13 @@ trait ApplicativeError[F[_], E] extends Applicative[F] {
    */
   def handleError[A](fa: F[A])(f: E => A): F[A] = handleErrorWith(fa)(f.andThen(pure))
 
+  /**
+   * Void any error, by mapping it to `Unit`.
+   *
+   * @see [[handleError]] to map to an `A` value instead of `Unit`.
+   */
+  def voidError[A](fa: F[A]): F[Unit] = handleError(void(fa))(_ => ())
```

Comment:
Alternative this could be `def voidError(fu: F[Unit]): F[Unit]` which may be used like `fa.void.voidError`. For some reason I thought that was awkward, but looking at it again it seems reasonable.

Reply:
Also I guess I should use `Function.const` here.

Reply:
I kind of like taking `F[Unit]` since if we have to call void on it then when A = Unit we are doing wasteful work.

It's a kind of false generalization. We can only accept any A by voiding so why not have the caller void.